plugins/com.contrastsecurity.ide.eclipse.core.tests/src/com/contrastsecurity/ide/eclipse/core/ContrastCacheTest.java

```diff
@@ -0,0 +1,114 @@
+package com.contrastsecurity.ide.eclipse.ui.test;
```

Comment:
Update package name. Should now be **com.contrastsecurity.ide.eclipse.core.tests**

Reply:
This class has been moved to its own fragment under UI plugin